thesis-gui/src/context/CategoryContext.tsx

```diff
@@ -36,8 +36,10 @@ export const CategoryContextProvider: FC = ({ children }) => {
     useEffect(() => {
         if (!user) {
```

Comment:
I would swap this if else:
```
if (user) {
  getCategories();
} else {
  setChosenCategories([]);
}
```

I think it's more readable when there is no negation in `if`.

Reply:
Done